cla-backend/cla/routes.py

```
@@ -1389,25 +1389,25 @@ def post_individual_signed_gitlab(
 
 
 @hug.post("/signed/gerrit/individual/{user_id}", versions=2)
-def post_individual_signed_gerrit(body, user_id: hug.types.uuid):
+def post_individual_signed_gerrit(content, user_id: hug.types.uuid):
     """
     POST: /signed/gerritindividual/{user_id}
 
     Callback URL from signing service upon ICLA signature for a Gerrit user.
     """
-    content = body.read()
+    content = request.bounded_stream.read()
```

Ruff (0.14.5)

Undefined name `request`

(F821)